### ISSUE TITLE
각자의견 및 본인이 짠 코드 설명(2018-04-09)

### ISSUE THREAD

**Issue by 1eedaegon** [1eedaegon/KYLius-method#26]
수원: 동영상 cnn까지 봄. 캐글 목차에 따라 역할분담 하면 어떨까. 데이터셋 준비, 정규화, cnn 등으로. 
상욱: 동영상 전부 봄. 오늘 코드 돌려봄. cnn 이미지 축소, 필터. 이미지 축소는 비슷하니까 필터를 늘리는게 정확도 향상에 가장 중요할 것 같음. 
대곤: mnist cnn 썼음. epoch 15로 돌림. conv2d, relu, maxpool, reshape, adam optimizer. session은 최적화 하려고 고치는 중. cnn이 값이 잘 나옴. 98.8%. 
승혁: 단순 mlp. (4개 layer). learning rate = 0.002, training epoch = 40, 각 레이어의 인풋과 아웃풋은 784->256->256->256->10. 정확도 94~96%.
우리 컴퓨터 맨 왼쪽꺼. 대곤이가 gpu셋팅 알려주기로 함.
앞으로 할거: 코드를 짜서 깃허브에 올리고 피드백. (4월 12일 목요일 밤까지)
cnn, rnn(lstm), (시간되면) mlp 
다음 회의 4월 13일 금요일 5시.
대곤이가 gpu셋팅 알려주는거: 이번주 중에 쉬는시간에 짬짬히.
